tools/print_pip_deps.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# A script to manage internal dependencies
+
+internal_deps=(
```

Comment:
Please add instructions on where these come from. Do we have a requirements file? Not sure if that would work